server/src/test/java/org/elasticsearch/cluster/coordination/CoordinatorTests.java

```diff
@@ -1096,41 +1151,43 @@ public void writeTo(StreamOutput out) throws IOException {
         public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
             return builder;
         }
+
     }
 
     public void testClusterRecoversAfterExceptionDuringSerialization() {
-        final Cluster cluster = new Cluster(randomIntBetween(2, 5)); // 1-node cluster doesn't do any serialization
-        cluster.runRandomly();
-        cluster.stabilise();
+        try (Cluster cluster = new Cluster(randomIntBetween(2, 5))) { // 1-node cluster doesn't do any serialization
```

Comment:
this is not true anymore. Perhaps we should amend this test.

Reply:
Good point, this is fine for 1-node clusters too now. Addressed in 92a02b7.